rust/src/cli/ncl.rs

```diff
@@ -280,10 +280,14 @@ fn main() {
 
         if let Some(file_paths) = matches.values_of("STATE_FILE") {
             let file_paths: Vec<&str> = file_paths.collect();
-            print_result_and_exit(
-                apply_from_files(&file_paths, matches),
-                EX_DATAERR,
-            );
+            if file_paths.get(0) == Some(&"-") {
```

Comment:
Instead of using `.get(0)`, use `.first()`.. this is causing a linter warning. Thank you!

Reply:
Fixed. Thanks!